sql-statements/sql-statement-kill.md

```diff
@@ -1,42 +1,63 @@
++---------------------+------+-----------------+-----------------------------------------------------------------------------+
+```
+
+{{< copyable "sql" >}}
+
+```sql
+KILL 5857102839209263511;
+```
+
+```
 Query OK, 0 rows affected (0.00 sec)
 ```
 
 ## MySQL compatibility
 
-* By design, `KILL` is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer.
-* DO NOT set [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query) in your configuration file UNLESS you are certain that clients will be always connected to the same TiDB node. This is because pressing <kbd>ctrl</kbd>+<kbd>c</kbd> in the default MySQL client opens a new connection in which `KILL` is executed. If there are proxies in between, the new connection might be routed to a different TiDB node, which possibly kills a different session.
-* The `KILL TIDB` statement is a TiDB extension. The feature of this statement is similar to the MySQL `KILL [CONNECTION|QUERY]` command and the MySQL command-line <kbd>ctrl</kbd>+<kbd>c</kbd> feature. It is safe to use `KILL TIDB` on the same TiDB node.
+- The `KILL` statement of MySQL can only terminate a connection in the currently connected MySQL instance, while the `KILL` statement of TiDB can terminate a connection in any TiDB instance in the entire cluster.
+- Currently, using the MySQL command line <kbd>ctrl</kbd>+<kbd>c</kbd> to terminate a query or connection in TiDB is not supported.
+
+## Behavior change descriptions
+
+Since v6.1.0, TiDB supports the Global Kill feature, which is enabled by default and controlled by the [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-new-in-v610-version) configuration. When the Global Kill feature is enabled, both the `KILL` and the `KILL TIDB` statement can terminate queries or connections across nodes so you do not need to worry about erroneously terminating queries or connections any more. When you use a client to connect to any TiDB node and execute the `KILL` statement or `KILL TIDB` statement, the statement will be forwarded to the corresponding TiDB node. If there is an agent between the client and TiDB, the `KILL` and `KILL TIDB` statements will also be forwarded to the corresponding TiDB node for execution.
+
+If the Global Kill function is not enabled or you are using a version earlier than TiDB v6.1.0, note the following:
+
+- By default, `KILL` is not compatible with MySQL. This helps prevent against a case of a connection being terminated by a wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer. To terminate other connections on the currently connected TiDB instance, you need to add the `TIDB` suffix explicitly by executing the `KILL TIDB` statement.
```

Comment:
behind?

Reply:
Where to use `behind`?

Reply:
"because it is common to place multiple TiDB servers behind a load balancer"